### PR TITLE
Fix navigation bug

### PR DESCRIPTION
**Pull request by Warhedgehog**
This fixes issue #1226. I also removed the back buttons of election and roll call setup pages since the back arrow offers the same functionality

### CODE REVIEW

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/election/fragments/ElectionSetupFragment.java [hunk 1]
```diff
@@ -94,39 +91,39 @@ public View onCreateView(
       @Nullable ViewGroup container,
       @Nullable Bundle savedInstanceState) {
 
-    mSetupElectionFragBinding = ElectionSetupFragmentBinding.inflate(inflater, container, false);
+    ElectionSetupFragmentBinding setupElectionFragBinding =
```
Comment (GabrielFleischer) [inline fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/election/fragments/ElectionSetupFragment.java:94]: In the rest of the app (where I renamed it too)I named the Binding `binding`. I feel like it is self-explanatory in the context, and it is enough.

What do you think ?

**Review by GabrielFleischer — APPROVED** [review]
Good job

We might need to do this fix at other places, but nothing comes to my mind right now. We'll create issues when necessary,

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/election/fragments/ElectionSetupFragment.java (hunk 1 above)
Comment (Warhedgehog) [reply fe2-android/app/src/main/java/com/github/dedis/popstellar/ui/detail/event/election/fragments/ElectionSetupFragment.java:94]: Much better I'll change it

**Comment by Warhedgehog** [comment]
I don't think it will problem because the conditions for it to happen are quite specific i.e. it has to be a different fragment than the Default tab but opened from the Default tab. Both `HomeActivity` and `DigitalCashActivity` offer no access to a different fragment.

**Review by GabrielFleischer — APPROVED** [review]
LGTM

**Pull request merged**